lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1130,9 +1130,103 @@ Error ONNXModelLoader::loadSlice(const ONNX_NAMESPACE::NodeProto &op,
   return Error::success();
 }
 
+Error ONNXModelLoader::loadConv1D(const ONNX_NAMESPACE::NodeProto &op,
+                                  ArgumentDictionaryTy &dict) {
+  const std::string &opName = loadOperatorName(op);
+  // Load the attributes
+  std::vector<glow::unsigned_t> strides(2, 1);
+
+  strides[1] = dict.count("strides") ? dict.at("strides")->ints(0) : 1;
+  strides[0] = 1;
+
+  unsigned_t group = 1;
+  if (dict.count("group")) {
+    ASSIGN_VALUE_OR_RETURN_ERR(group, loadInt(dict.at("group")));
+  }
+
+  unsigned_t dilation =
+      dict.count("dilations") ? dict.at("dilations")->ints(0) : 1;
+  // Load the inputs
+  NodeValue in;
+  // input == NCW ---> NCHW
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+  in = G_->createExpandDims(opName, in, 2);
+  // filetervalue == CKS ---> CKRS
+  NodeValue filterValue;
+  ASSIGN_VALUE_OR_RETURN_ERR(filterValue, getNodeValueByName(op.input(1)));
+  filterValue = G_->createExpandDims(opName, filterValue, 2);
+  // Transpose the filter to the right format. Glow expects to read the
+  // weights in the format CRSK. ONNX stores the operators as CKRS.
+  // C - output_depth, R - filter_height, S - filter_width, K - input_depth.
+  // filetervalue == CKRS ---> CRSK
+  TransposeNode *filterTransposeNode =
+      G_->createTranspose(opName, filterValue, {0u, 2u, 3u, 1u});
+  // The structure of the conv weights is: CRSK. We take the C, which is the
+  // number of filters. We use this value to calculate the size of the bias
+  // if it is not specified.
+  const NodeValue filterTransposedValue = filterTransposeNode->getResult();
+  dim_t depth = filterTransposedValue.dims()[0];
+  // Get the kernel shape from the input.
+  llvm::SmallVector<unsigned_t, 2> kernelShape(2);
+  kernelShape[0] = filterTransposedValue.dims()[1];
+  kernelShape[1] = filterTransposedValue.dims()[2];
+
+  // Construct the Bias field.
+  Constant *bias = nullptr;
+
+  // Check if we have a serialized bias vector.
+  if (op.input_size() > 2) {
+    auto &biasTensorName = op.input(2);
+    // Load the serialized bias vector.
+    ASSIGN_VALUE_OR_RETURN_ERR(bias, getConstantByName(biasTensorName));
+  }
+
+  // If a serialized bias wasn't found then create a zero bias.
+  if (!bias) {
+    Tensor biasTensor(ElemKind::FloatTy, {depth});
+    biasTensor.zero();
+    bias = G_->getParent()->createConstant("conv.bias", std::move(biasTensor));
```

Comment:
nit: I see this was copied from elsewhere, but wondering if we can just create a Splat for this instead of materializing it as a Constant so early. We'd change `bias` to be a `NodeValue` in this case.

Reply:
@jfix71 reference code of loadConv was taken for loadConv1D implementation and leaving the code changes as is. If this is really required we can revisit. Or i will raise another PR correcting both loadConv and loadConv1D APIs